indico/modules/rb_new/client/js/common/timeline/TimelineItem.module.scss

```diff
@@ -145,3 +150,31 @@
 .popup-center {
     text-align: center;
 }
+
+:global(.room-details-usage),
+:global(.book-room-ocurrences-timeline) {
+    .timeline-occurrence {
+        &:global(.booking),
+        &:global(.pre-booking),
+        &:global(.default) {
+            cursor: default;
+        }
+    }
+}
+
+:global(.book-room-timeline-view) {
+    .timeline-occurrence {
+        &:global(.booking),
+        &:global(.pre-booking) {
+            cursor: default;
+        }
+    }
+}
+
+:global(.unavailable-rooms-modal),
```

Comment:
is there a better way than using global class names? e.g. checking which types of click handlers are actually defined and then using that info to determine which bars are clickable

Reply:
Don't know what I was thinking of :man_facepalming: Thanks!